content/v2/openapi.yml

```diff
@@ -3608,6 +3608,8 @@ components:
           type: boolean
         whois_privacy:
           type: boolean
+        linked_provider:
+          type: string
```

Comment:
I recommend we set the same description also here. I believe the field should have some sort of summary or description field.

Reply:
Thanks! I will move this as this was in the wrong place.